client/js/screens/AboutUs/styles.js

```diff
@@ -0,0 +1,48 @@
+import {StyleSheet} from 'react-native';
+import {THEME} from '../../config';
+const {
+  colors: {black, darkGrey},
+  formatting: {baseSpacing, centeredChildren},
+  typography: {headerFont, hugeText, contentText, mainFont, subHeaderText},
+} = THEME;
+
+const styles = StyleSheet.create({
+  page: {...centeredChildren},
+  container: {
+    ...centeredChildren,
+    width: 308,
```

Comment:
the container should not be given a fixed width. Use % if needed or use flex properties.